packages/rocketchat-lib/server/functions/setEmail.js

```diff
@@ -1,8 +1,7 @@
 RocketChat._setEmail = function(userId, email) {
 	email = s.trim(email);
-	if (!userId) {
+	if (!userId)
 		throw new Meteor.Error('invalid-user', '[methods] setEmail -> Invalid user');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'throw'.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2317368006/issues/source?bid=3199634&fileBranchId=3203539#l4)